ush/config.sh.RRFS_dev1

```diff
@@ -21,12 +21,12 @@ FIX_GSI=/mnt/lfs4/BMC/nrtrr/RRFS/fix/fix_gsi
 FIX_CRTM=/home/rtrr/FIX_EXEC_MODULE/crtm/CRTM_v2.3.0
 OBSPATH_NSSLMOSIAC=/public/data/radar/nssl/mrms/conus
 RADARREFL_TIMELEVEL=(0 15 30 45)
-#FH_DFI_RADAR="0.0,0.25,0.5"
+FH_DFI_RADAR="0.0,0.25,0.5"
 
 VERBOSE="TRUE"
 
 RUN_ENVIR="nco"
-PREEXISTING_DIR_METHOD="rename"
+PREEXISTING_DIR_METHOD="upgrade"
```

Comment:
Should this not be done for all configs? Seems that we want to have as much consistency across configs as possible for smooth deployment.

Reply:
We are trying to deploy dev1. I would like to change others when we need to deploy others. This can be a PR from other colleagues. Just want to make RRFS_dev1 deploy simple as we need to catch 22Z cold start.

Reply:
We need to keep these all consistent, especially in a space where everyone is learning. If the proper way to deploy a cycling run and do it smoothly is to use "upgrade" then all the cycling configs should be modified to that now.

Our other colleagues are still learning these things (including me). We were taught to use this setting in the last few sessions as the trusted method, so it should be the default to get a cycling method to run smoothly. Please go ahead and update them.